keywords/SyncGateway.py

```diff
@@ -278,6 +278,7 @@ def start_sync_gateways(self, cluster_config, url=None, config=None):
             "sg_cert_path": sg_cert_path,
             "server_port": self.server_port,
             "server_scheme": self.server_scheme,
+            "logging": "",
```

Comment:
no need to add it over here, as it is going to be added in either if or else

Reply:
done